core/src/main/java/org/apache/carbondata/core/indexstore/BlockletDataMapIndexStore.java

```diff
@@ -49,6 +50,7 @@
    */
   protected CarbonLRUCache lruCache;
 
+  Map<String, Map<String, BlockMetaInfo>> segInfoCache;
```

Comment:
What is this used for?

Reply:
It's used for reduce the S3 IO, It needs 70*140 IO before, now it only need 140 IO